source/content/overview/core.md

```diff
@@ -0,0 +1,30 @@
+# Brightway core libraries
+
+The four core libraries in Brightway are:
+
+| Project | Description |
+| ------- | ----------- |
+| [`bw2data`](https://github.com/brightway-lca/brightway2-data) | Project management, graph storage and ORM using SQLite, search |
+| [`bw2calc`](https://github.com/brightway-lca/brightway2-calc) | Matrix-based calculations |
+| [`bw2io`](https://github.com/brightway-lca/brightway2-io) | [ETL](https://en.wikipedia.org/wiki/Extract,_transform,_load) for LCA data |
+| [`bw2analyzer`](https://github.com/brightway-lca/brightway2-analyzer) | Analyze inventory data and LCA results |
+
+These core libraries in turn build on the following support libraries:
```

Comment:
...a discussion perhaps better included in [the 'Software Framework' page](https://docs.brightway.dev/en/latest/content/other/framework.html), which I hope to expand/rename during Brightcon?

Reply:
It might make sense to move the content from the 'Software Framework' page to this new `core.md`.

The idea is to give a nice overview of the entire software ecosystem, including a to-be-designed Mermaid diagram based on [Romain's draw.io illustration](https://github.com/brightway-lca/brightway-documentation/blob/37f192843e784ed5106910a1b7246c9ed3092358/source/content/gettingstarted/_images/brightway_ecosystem_diagram.svg?plain=1#L3).

Thoughts, @cmutel?

Reply:
Maybe we can fold these two together? I think that "Other" is going to get a lot less attention...

Reply:
Yes, that makes sense. You can leave the 'Software Framework' page be for now - I will merge it into your new 'Overview' section at Brightcon.